code/__DEFINES/~skyrat_defines/inventory.dm

```diff
@@ -4,6 +4,7 @@
 #define STYLE_TAUR_SNAKE (1<<2) //taur-friendly suits
 #define STYLE_TAUR_PAW (1<<3)
 #define STYLE_TAUR_HOOF (1<<4)
-#define STYLE_TAUR_ALL (STYLE_TAUR_SNAKE|STYLE_TAUR_PAW|STYLE_TAUR_HOOF)
+#define STYLE_TAUR_ALL (STYLE_TAUR_SNAKE | STYLE_TAUR_PAW | STYLE_TAUR_HOOF)
+#define STYLE_TAUR (STYLE_TAUR_SNAKE & STYLE_TAUR_PAW & STYLE_TAUR_HOOF)
```

Comment:
Move that down from the bitfield, and it might be wise to make the first one more descriptive, or add some AUTODOC documentation to it.